spec/requests/api/internal/v2/court_applications_request_spec.rb

```diff
@@ -9,10 +9,49 @@
   let(:court_application_id) { SecureRandom.uuid }
 
   describe "GET /api/internal/v2/court_applications/{court_application_id}" do
-    it "return 200 succeess" do
+    before do
+      stub_request(:get, "#{ENV['COMMON_PLATFORM_URL']}/applications/#{court_application_id}")
+        .to_return(
+          status: status,
+          headers: { content_type: "application/json" },
+          body: body,
```

Comment:
is rubocop happy with this? 🤔 

I though it wants only:
`body:`

Reply:
Good spot. Rubocop didn't complain (I'm kinda suprised we don't have rubocop in our CI pipeline??), but you're right, it's neater to remove the repetition.

Reply:
I'm surprised too, I think it used to complain, we should check it, anyway it's ok for now!

Reply:
Oh I'm being silly. It _is_ in the CI pipeline, it's just buried in the `test` workflow.